tests/integration/test_models/test_full_battery_models/test_lead_acid/test_compare_outputs.py

```diff
@@ -92,6 +92,25 @@ def test_compare_outputs_surface_form(self):
             comparison = StandardOutputComparison(solutions)
             comparison.test_all(skip_first_timestep=True)
 
+    def test_compare_outputs_timescale(self):
```

Comment:
This works well for SPM but if you run DFN you see differences in solutions. Is that just because of solver?